components/calendar/Cargo.toml

```diff
@@ -6,7 +6,7 @@
 name = "icu_calendar"
 description = "API for supporting various types of calendars"
 version = "1.2.0"
-rust-version = "1.61.0"
+rust-version = "1.65.0"
```

Comment:
Issue/Discussion: did we agree to eagerly update the Cargo.toml files? My instinct would be to keep them as low as possible and bump them on a crate-by-crate basis when needed, in compliance with the MSRV policy.

Reply:
I guess we don't have a way to test that each individual crate compiles on its stated msrv? Does cargo publish do that for us?

Reply:
No, and I think it's should homogenize if we aren't testing it

Reply:
Also note that for soundness reasons I'm really hoping to get `slice_from_raw_parts()` into zerovec and as a foundational crate that everything depends on that kinda sets our floor to 1.64 anyway.

(It _is_ a soundness fix we could defer if we would like: the safety model isn't completely pinned down yet and it's unlikely to get exploited, but I'd like to really harden our bigger unsafe crates well before it's necessary)

Reply:
How hard is the rust version in Cargo.toml? Will cargo refuse to compile or is it just a warning?

Reply:
Refuse to compile.